crates/ra_hir_ty/src/traits/chalk.rs

```diff
@@ -183,6 +183,8 @@ impl<'a> chalk_solve::RustIrDatabase<Interner> for ChalkContext<'a> {
                     .collect(),
                 1,
             ),
+            // FIXME: fill these in
```

Comment:
r? @flodiebold

Reply:
Return-position impl Trait can't have where clauses, and we don't support other kinds of opaque types yet, so this should be fine.